src/components/item-info/index.js

```diff
@@ -37,7 +43,7 @@ export const ItemInfo = ({
               <Primary>{walletPreview(token_info.creators[0])}</Primary>
             </Button>
           </div>
-          {!notForSale && <p>Edition: {edition}</p>}
```

Comment:
What does it show on the button? Would it be nice to see the editions (10/10) (so you know how many were minted) and on the button see SOLD OUT and have it disabled?